framework/k4SimDelphes/examples/options/edm4hep_output_config.tcl

```diff
@@ -0,0 +1,12 @@
+module EDM4HepOutput EDM4HepOutput {
+    add ReconstructedParticleCollections EFlowTrack EFlowPhoton EFlowNeutralHadron
+    add GenParticleCollections           Particle
+    add JetCollections                   Jet
+    add MuonCollections                  Muon
+    add ElectronCollections              Electron
+    add PhotonCollections                Photon
+    add MissingETCollections             MissingET
+    add ScalarHTCollections              ScalarHT
+    set RecoParticleCollectionName       ReconstructedParticles
+    set MCRecoAssociationCollectionName  MCRecoAssociations
+ }
```

Comment:
Is it possible to ship only one copy of this? This seems to add a second copy to the one that is already present in `examples`, right? Maybe one of the two can be removed.